app/controllers/admin/tickets_controller.rb

```diff
@@ -34,22 +34,34 @@ def create
     end
 
     def show
-      @ticket = SupportTicket.find(params[:id])
+      ticket
     end
 
     def resolve
-      @ticket = SupportTicket.find(params[:id])
+      ticket.update ticket_update_params.merge(closed_at: Time.zone.now)
 
-      if @ticket.close!(current_user)
+      if ticket.close!(current_user)
```

Comment:
Looks like `close!` is already setting the `closed_at` time. Instead of updating the ticket twice, you might try passing the `closer` (current_user) and the `closer_notes` to the `close!` method.

Reply:
resolved in https://github.com/rubyforgood/inkind-admin/pull/183